mm2src/coins/nft.rs

```diff
@@ -3,6 +3,7 @@ use mm2_err_handle::prelude::{MmError, MmResult};
 
 pub(crate) mod nft_errors;
 pub(crate) mod nft_structs;
+#[cfg(any(test, target_arch = "wasm32"))] mod nft_tests;
```

Comment:
I think target check isn't needed here

Reply:
Lets have it, I didnt see moralis tests in wasm without it

Reply:
This is because we are not using `cargo test` in CI. Not so important, but it would make more sense to update those to `cargo test` to recognize `test` flags